charts/sophora-seo-check/templates/pvc.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+
+kind: PersistentVolumeClaim
+
+metadata:
+  name: {{ include "sophora-seo-check.fullname" . }}-data
```

Comment:
As per your recommendation, I have converted it to a StatefulSet, along with a volume claim template.
The app container needs a volume to store data persistently.

I have removed pvc.yaml since it is no longer needed.